src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -644,6 +644,7 @@ export class InputDatePicker
 
   transitionEl: HTMLDivElement;
 
+  @Watch("open")
```

Comment:
@jcfranco the problem seems to be when the input-date-picker is rendered the `floatingEl` has no parent yet. Not sure if its because it's not rendered or if some other stuff isn't ready yet.

So if we watch for `open` and then setup the floating ui again it can get all the scrolling parents correctly.

Seems like this is the only component affected.